tests/update/updates_packagekit_kde.pm

```diff
@@ -37,10 +37,15 @@ sub run {
         while (1) {
             assert_and_click_until_screen_change('updates_click-install');
 
+            # Wait until installation starts, intended to time out
+            wait_still_screen(stilltime => 4, timeout => 5);
```

Comment:
Doesn't this destroy the proper checks in L45-L59 checking for https://bugzilla.suse.com/show_bug.cgi?id=992773 and https://bugzilla.suse.com/show_bug.cgi?id=1041112 ?

Reply:
x86_64: https://openqa.opensuse.org/tests/613272/file/autoinst-log.txt
The following assert_screen takes ~23 seconds until it matches. Inserting the wait_still_screen only delays the first assert_screen attempt by 5 seconds, so it still tries to match early enough.
Theoretically, the update could finish in less than 5 seconds. In this case the glitches might go unnoticed, but a) I don't think the update is so fast, and b) the opencv execution time may also cause frame drops, hiding the glitches.

Reply:
I might be wrong, but wait_still_screen expects that screen won't change for 4 seconds. But after clicking install button the update process will start and it's showing progress - so screen is changing. Isn't this always failing?

Reply:
"..., intended to time out"

It recurrently checks for screen changes, it does it for 5 seconds (until the time out). After this 5 seconds, we can be quite sure any further changes are the update process, *not* the stepwise UI updates after the mouse click.

the assert_and_click_until_screen_change asserts there had been a UI change after the click, but it cant tell apart the redrawing of the clicked button from the update of the UI text ("You have 7 new updates" -> "Getting updates").

On fast machines, the UI is *typically* already updated completely when the assert_screen runs, but especially on aarch64 you can see the UI update in distinct steps.

Reply:
Why not just use `sleep 5` if it's intended to time out? (I originally thought it will die on timeout by mistake).

Reply:
Debugging - with sleep 5, there is no feedback if/when the screen changes.

Reply:
@okurz - seems those bugs are happily ignored (not important) - I merged because I don't think they need more attention from our side